packages/dev/gui/src/2D/controls/container.ts

```diff
@@ -142,6 +142,7 @@ export class Container extends Control {
 
     public _flagDescendantsAsMatrixDirty(): void {
         for (const child of this.children) {
+            child._isClipped = false;
             child._markMatrixAsDirty();

```

Comment:
Shouldn t the marMatrixAsDirty set the flag ??? just wondering :-)

Reply:
yeah, this is a very special case! I was struggeling with this decision.
This special case appears ONLY when a container has a container as a child. Setting isClipped every time the matrix is set dirty will be redundant. This is the least amount of isClipped-resetting.
But I agree with you that it does feel a bit off.

Reply:
maybe let s wait for @deltakosh feedback tomorrow as I remember the caching was pretty heavy to put in place :-)

Reply:
I am anyhow not merging this until approved :-)